ops/manifests/manifest.py

```diff
@@ -112,9 +94,8 @@ def __init__(
     @cached_property
     def client(self) -> Client:
         """Lazy evaluation of the lightkube client."""
-        with _proxied_env():
-            # Construct the client with juju proxy environment vars
-            client = Client(field_manager=f"{self.model.app.name}-{self.name}")
+        # Construct the client with juju proxy environment vars
```

Comment:
I think we should remove this comment as well

Reply:
🦖